docfx_yaml/monkeypatch.py

```diff
@@ -408,7 +409,8 @@ def transform_all(self, node):
                     child_copy.pop(0)
                     data['example'] = transform_node(child_copy)
                 else:
-                    content = transform_node(child)
+                    child_copy = child.deepcopy()
```

Comment:
Is there any reason we should copy the node?

Reply:
The node is modified and the original content is lost which caused some errors.